introduction.md

```diff
@@ -2,3 +2,6 @@ This is the introduction file for dojo_rules
 User: bevett
 Code School GitHub course
 Favorite code school path: Ruby
+*Git
+*JavaScript
+*Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?